cpp/dolfin/function/FunctionSpace.cpp

```diff
@@ -186,6 +186,7 @@ void FunctionSpace::interpolate(
                                          1, std::multiplies<>());
   Eigen::Array<PetscScalar, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>
       values(x.rows(), value_size);
+  values = 0;
```

Comment:
This shouldn't be needed.

Reply:
I just realized that the error was showing up because my `interpolate` function was not setting all the values of `values`. Now that I fixed that, the error no longer shows up. Feel free to decline this PR if this case is not relevant.